src/lib/maplibre/layer_helper.ts

```diff
@@ -0,0 +1,33 @@
+import type { Map } from "maplibre-gl";
+
+export class LayerHelper {
+  id: string;
+  defaultPaintValues: Array<[string, string]>;
+  changedValues: Array<string> = [];
```

Comment:
If our approach is to scrape everything initially, then we can remove this -- it's not doing anything right now